test/hubble/hubble_test.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	k8s "github.com/Azure/azure-container-networking/test/integration"
+	"github.com/Azure/azure-container-networking/test/internal/retry"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+const (
+	retryAttempts = 10
+	retryDelay    = 5 * time.Second
+	promAddress   = "http://localhost:9965/metrics"
+	labelSelector = "k8s-app=cilium"
+	namespace     = "kube-system"
+)
+
+var (
+	defaultRetrier  = retry.Retrier{Attempts: retryAttempts, Delay: retryDelay}
+	requiredMetrics = []string{
+		"hubble_flows_processed_total",
+		"hubble_tcp_flags_total",
+	}
+)
+
+func TestEndpoints(t *testing.T) {
+	var kubeconfigPath string
+	flag.StringVar(&kubeconfigPath, "kubeconfig", getDefaultKubeconfigPath(), "Path to the kubeconfig file")
+	flag.Parse()
+
+	config, err := getClientConfig(kubeconfigPath)
```

Comment:
nit: We have the `kubernetes` package that provide k8s api calls. https://github.com/Azure/azure-container-networking/tree/master/test/internal/kubernetes. 

The function that could replace this would be https://github.com/Azure/azure-container-networking/blob/0b45d15788d8f7c727f1adca95c11c9261dbefc1/test/internal/kubernetes/utils.go#L54-L60

Reply:
true, I forgot these were here :)